public/app/features/dashboard/components/Inspector/InspectHeader.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC } from 'react';
+import { css } from 'emotion';
+import { Icon, stylesFactory, Tab, TabsBar, useTheme } from '@grafana/ui';
+import { GrafanaTheme, SelectableValue } from '@grafana/data';
+import { InspectTab } from './PanelInspector';
+import { PanelModel } from '../../state';
+
+interface Props {
+  tab: InspectTab;
+  tabs: Array<{ label: string; value: InspectTab }>;
+  stats: { requestTime: number; queries: number; dataSources: number };
+  panel: PanelModel;
+  isExpanded: boolean;
+
+  onSelectTab: (tab: SelectableValue<InspectTab>) => void;
+  onClose: () => void;
+  onToggleExpand: () => void;
+}
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  return {
+    header: css`
+      background-color: ${theme.colors.formInputBg};
```

Comment:
should this not be pageHeaderBg ?

Reply:
Nope, pageHeaderBg is a gradient. This should be a solid color (gray15 for dark and not sure what for light, but probably white)

Reply:
ok, posted video in #ux to see if we should change page header then to make it consistent with this design